rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -2396,3 +2396,35 @@ rocalNop(RocalContext p_context,
     }
     return output;
 }
+
+RocalTensor ROCAL_API_CALL
+rocalPreEmphasisFilter(RocalContext p_context,
+                       RocalTensor p_input,
+                       RocalTensorOutputType rocal_tensor_output_datatype,
+                       bool is_output,
+                       RocalFloatParam p_preemph_coeff,
+                       RocalAudioBorderType preemph_border_type) {
+    if(!p_context || !p_input)
+        THROW("Null values passed as input")
+    Tensor* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    auto preemph_coeff = static_cast<FloatParam*>(p_preemph_coeff);
+    RocalTensorDataType op_tensorDataType;
```

Comment:
Please change the  op_tensorDataType to snake case output_tensor_datatype
Also move the declaration inside try catch to reduce the scope